public/map-controller/map-controller.js

```diff
@@ -213,21 +232,23 @@ var MapController = P({
    */
   post_initial_load: function() {
     var distance_fcn = make_distance_from_viewport_center(this.map.getBounds());
-    var features_by_distance = _.sortBy(this.region_details.get_geojson_features(), distance_fcn);
-
-    var sequence = Q(null).then(
-      this.load_feature_chunk.bind(this, _.take(features_by_distance, 500))
-    ).then(function() {
-      this.loading_status.setLoadedTopojson();
-    }.bind(this));
-
-    _.forEach(_.chunk(_.drop(features_by_distance, 500), 2500), (function(feature) {
-      sequence = sequence.delay(10).then(this.load_feature_chunk.bind(this, feature));
-    }).bind(this));
-
-    sequence.fail(function(err) {
-      console.error(err);
-    });
+    var add_country = (function(country_code) {
+      var features_by_distance = _.sortBy(this.admin_details.get_geojson_features(country_code),
+                                          distance_fcn);
+
+      var sequence = Q(null).then(
```

Comment:
I think I did what you meant.

Though now that I look at this again, it seems better to just do all countries' polygons together instead of separately. Added a TODO.